x-pack/plugins/ingest_manager/common/services/config_to_yaml.ts

```diff
@@ -10,6 +10,7 @@ const CONFIG_KEYS_ORDER = [
   'id',
   'name',
   'revision',
+  'dataset',
```

Comment:
Should this be `data_stream`? I think I miss how this works.

Reply:
this is just for ordering of yaml keys in the UI. I added this so that `data_stream.dataset` displays above `data_stream.type`

![image](https://user-images.githubusercontent.com/1965714/89455173-bb4f2500-d716-11ea-9c1c-290bcf721660.png)